Source.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+
+int main() {
+	float x, y;
+	char a;
+	std::cout << "use the formula (x + y)(x - y)(x * y)(x / y)" << std::endl;
+	for (int i = 1; i < 10; i++)
```

Comment:
can you make it so, that the program does not end with ten operations, but calculates the result and expects another operation until the user ends it, e.g with typing "end"?